contracts/Operator.sol

```diff
@@ -109,33 +94,29 @@ contract Operator is Upgradeable {
     );
   }
 
-  function vote(uint256 _proposalId) public coreOnly returns (uint _pId, bool _executed) {
-    var p = proposals[_proposalId];
+  function vote(uint256 proposalId) public coreOnly {
+    var p = proposals[proposalId];
     require(!p.executed);
-    uint256 contributorId = contributorsContract().getContributorIdByAddress(msg.sender);
-    require(p.votes[contributorId] != true);
-    p.voterIds.push(contributorId);
-    p.votes[contributorId] = true;
+    uint256 voterId = contributorsContract().getContributorIdByAddress(msg.sender);
```

Comment:
Why the change from `contributorId` -> `voterId`?

Reply:
yeah...hmmm.. thought it is more descriptive.. but I see your point.